src/condor_utils/file_transfer.cpp

```diff
@@ -253,7 +254,7 @@ FileTransfer::SimpleInit(ClassAd *Ad, bool want_check_perms, bool is_server,
 
 	dprintf(D_FULLDEBUG,"entering FileTransfer::SimpleInit\n");
 
-	/* in the case of SimpleINit being called inside of Init, this will
+	/* in the case of SimpleInit being called inside of Init, this will
```

Comment:
In the future, please split out correction / whitespace changes into a separate patch (`git add --patch` allows you to easily isolate hunks like this).